modules/admin-ui/src/main/webapp/scripts/shared/directives/editableSingleSelectDirective.js

```diff
@@ -124,21 +126,30 @@ angular.module('adminNg.directives')
         $timeout.cancel(scope.submitTimer);
       });
 
+      scope.toggleEditMode = function () {
+
+        if (scope.editMode == false) {
+          scope.enterEditMode();
+        } else {
+          scope.leaveEditMode();
+        }
+      };
+
       scope.enterEditMode = function () {
+
         // Store the original value for later comparision or undo
         if (!angular.isDefined(scope.original)) {
           scope.original = scope.params.value;
         }
         scope.editMode = true;
-        scope.focusTimer = $timeout(function () {
+        $timeout(function () {
           if ($('[chosen]')) {
-            element.find('select').trigger('chosen:activate');
+            element.find('select').trigger('chosen:open');
           }
         });
       };
 
       scope.leaveEditMode = function () {
-        // does not work currently, as angular chose does not support ng-blur yet. But it does not break anything
```

Comment:
Unfortunately, this still holds, although the comment is not correct: It is jquery-chosen that makes it impossible for angular-chosen to implement ng-blur.
Your idea to focus the pencil icon instead of the chosen dropdown would make this very, very nasty bug less important, but it still would exist: Left-click a chosen dropdown and then press the tab key.